common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -361,6 +391,84 @@ def test_can_reset_attempts(self):
         exam_attempts_section.remove_student_attempt()
         self.assertFalse(exam_attempts_section.is_student_attempt_visible)
 
+    @attr('a11y')
+    def test_track_selection_a11y(self):
+        """
+        Proctored: Track selection accessibility tests
+        """
+        self.track_selection_page.a11y_audit.config.set_rules({
+            "ignore": [
+                'document-title',
+                'html-lang',
+            ]
+        })
+        self.track_selection_page.a11y_audit.check_for_accessibility_errors()
+
+    @attr('a11y')
+    def test_payment_verification_a11y(self):
+        """
+        Proctored: Payment verification accessibility tests
+        """
+        self.payment_and_verification_flow.a11y_audit.config.set_rules({
+            "ignore": [
+                'document-title',
+                'html-lang',
+            ]
+        })
+        self.payment_and_verification_flow.a11y_audit.check_for_accessibility_errors()
+
+    @attr('a11y')
+    def test_immediate_verification_a11y(self):
+        """
+        Proctored: Immediate verification accessibility tests
+        """
+        self.immediate_verification_page.a11y_audit.config.set_rules({
+            "ignore": [
+                'document-title',
+                'html-lang',
+            ]
+        })
+        self.immediate_verification_page.a11y_audit.check_for_accessibility_errors()
+
+    @attr('a11y')
+    def test_upgrade_a11y(self):
+        """
+        Proctored: Upgrade page accessibility tests
+        """
+        self.upgrade_page.a11y_audit.config.set_rules({
+            "ignore": [
+                'document-title',
+                'html-lang'
+            ]
+        })
+        self.upgrade_page.a11y_audit.check_for_accessibility_errors()
+
+    @attr('a11y')
+    def test_fake_payment_a11y(self):
+        """
+        Proctored: Fake payment page accessibility tests
+        """
+        self.fake_payment_page.a11y_audit.config.set_rules({
+            "ignore": [
+                'document-title',
+                'html-lang',
+            ]
+        })
+        self.fake_payment_page.a11y_audit.check_for_accessibility_errors()
+
+    @attr('a11y')
```

Comment:
Delete this test and self.problem_page-- I don't see that self.problem_page is ever referenced (certainly no "problem" is being visited).